### ISSUE TITLE
GROWS

### ISSUE THREAD

**Issue by harkirats043** [harkirats043/Project3#2]
One area of code that could have be improved is when loading the dog and fox pictures I have added a detail page to show the information(details) regarding the particular image.

**Comment by krabecb** [comment]
Yes, I agree that a detail page would be an excellent icebox feature!
